config-linux.md

```diff
@@ -686,6 +703,45 @@ The following parameters can be specified to set up seccomp:
 }
 ```
 
+### <a name="containerprocessstate" />The Container Process State
+
+The container processs state is a data structure passed via a UNIX socket.
```

Comment:
typo in `processs`

Reply:
nice catch, fixed locally. Will push soon!